app/controllers/sessions_controller.rb

```diff
@@ -1,19 +1,25 @@
 class SessionsController < ApplicationController
+  before_action :load_user, only: :create
+
   def new; end
 
   def create
-    user = User.find_by email: params[:session][:email].downcase
-    if user && user.authenticate(params[:session][:password])
-      log_in user
-      redirect_to user
+    if @user && @user.authenticate(params[:session][:password])
```

Comment:
Sao dùng biến instance vậy em
Dùng biến local đi

Reply:
Nếu dùng biến local thì em phải find cái user object trong hàm create luôn
Rubocop bắt lỗi "Assignment Branch Condition size for create is too high".
Nên em tách ra hàm load_user để tìm user rồi dùng before_action. Nên phải khai báo biến instance user thay vì local